src/frontend/src/styles/_overrides.scss

```diff
@@ -0,0 +1,15 @@
+@use '@carbon/react/scss/breakpoint' as *;
+
+.cds--content {
+    margin-top: 3rem;
+    padding: 0;
+    background: var(--cds-background);
+}
+
+.cds--content .cds--css-grid {
+    max-width: 100%;
+    @include breakpoint(md) {
+        padding-left: 20px;
+        padding-right: 20px;
+    }
+}
```

Comment:
I've added this globally.

This overrides the cds design for grids to make them expand to the entire page width